index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alimasa Eliza Page</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+  <body>
+    <div class="header">
+      <h1 class="head">Introduction</h1>
+    </div>
+    <hr />
+    <div>
+      <div class="image">
+        <img class="pfp" src="pfp.jpg" alt="Profile Picture" />
+        <h1 class="heading">Alimasa Eliza</h1>
+        <p class="description">Student at PUP-Taguig taking DIT course.</p>
+      </div>
+    </div>
+    <hr />
+    <div class="content-container">
+      <div class="content">
+        <h2 class="info">My Interests and Hobbies</h2>
+        <hr />
+        <ul class="info-li">
+          <li>Playing online games</li>
+          <li>Listening to music</li>
+          <li>Sleeping</li>
+          <li>Reading</li>
+          <li>Cats</li>
+        </ul>
+      </div>
+      <hr />
+      <div class="content">
+        <h2 class="files">My Works</h2>
+        <hr />
+        <ol class="files-li">
+          <li><a href="./test.html">Test</a></li>
+          <li><a href="./test1.html">Test 1</a></li>
+          <li><a href="./test2.html">Test 2</a></li>
+          <li><a href="./test3.html">Test 3</a></li>
+          <li><a href="./test4.html">Test 4</a></li>
+        </ol>
+      </div>
+    </div>
+    <hr />
+    <div class="content">
+      <h2 class="testi">Testimonials</h2>
+      <hr />
+      <ul class="testi-li">
+        <li>
+          I find this website to be great. Its user-friendly interface. -
+          Mikaella
+        </li>
+        <li>
+          Creative and Beautiful website. It exceeded my expectations. Good Job!
+          - Sofia
+        </li>
+        <li>
+          Her attention to detail and dedication to excellence were evident in
+          every aspect of her website. - Alona
+        </li>
+        <li>
+          Creativity and dedication shine through in every project. A very good
+          work for you! - Precious
+        </li>
+      </ul>
+    </div>
+    <hr />
+    <div class="content">
+      <h2 class="goals">My Goals</h2>
+      <hr />
+      <ul class="goals-li">
+        <li>Gain Weight</li>
+        <li>Travel the world</li>
+        <li>Start a business</li>
+        <li>Build my self confidence</li>
+        <li>Improve and develop skills</li>
+        <li>Provide the needs of stray animals</li>
+      </ul>
+    </div>
+    <hr />
+    <div>
+      <h2 class="remarks">Comment Section</h2>
+      <hr />
+      <div class="content">
+        <div>
+          <button onclick="sortComments('asc')">Oldest</button>
+          <button onclick="sortComments('desc')">Newest</button>
+        </div>
+        <ul class="com-li" id="commentList"></ul>
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. 
See the reference for clarifications. 
```suggestion
        <ul class="com-li" id="comment_list"></ul>
```

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5